src/main/java/uk/gov/companieshouse/charges/data/controller/ChargesController.java

```diff
@@ -51,4 +53,34 @@ public ChargesController(final ChargesService chargesService, final Logger logge
         return ResponseEntity.status(HttpStatus.OK).build();
     }
 
+    /**
+     * Retrieve a company charge details using a company number and chargeId.
+     *
+     * @param companyNumber the company number of the company
+     * @param chargeId      the chargeId
+     * @return company charge api
+     */
+    @GetMapping("/company/{company_number}/charge/{charge_id}")
```

Comment:
I think it should be plural /charges instead of /charge, sorry, maybe I got it wrong on ticket

Reply:
reverted to plural /charges thanks